client/components/confirm-modal/styles.scss

```diff
@@ -1,7 +1,7 @@
 @import "@automattic/color-studio/dist/color-variables";
 @import "@automattic/typography/styles/variables";
 
-.unsubscribe-modal {
+.confirm-modal {
```

Comment:
Considering this will become part of global scope for entire calypso, it feels little too generic CSS class name and could conflict with other UIs.